src/elements/content-uploader/UploadStateContent.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import { FormattedMessage } from 'react-intl';
+import UploadInput from './UploadInput';
+import messages from '../common/messages';
+
+export interface UploadStateContentProps {
+    fileInputLabel?: React.ReactNode;
+    folderInputLabel?: React.ReactNode;
+    message?: string;
+    onChange?: Function;
+    useButton?: boolean;
+}
+
+const UploadStateContent = ({
+    fileInputLabel,
+    folderInputLabel,
+    message,
+    onChange,
+    useButton = false,
+}: UploadStateContentProps) => {
+    const inputLabelClass = useButton ? 'btn btn-primary be-input-btn' : 'be-input-link'; // TODO: Refactor to use Blueprint components
```

Comment:
i created a ticket to handle this as a fast follow. right now, we're applying these classes to mirror the BUIE component but instead we should directly use the blueprint components. it mainly requires refactoring the UploadInput component which is out of scope for this PR